example/lib/grid/image_grid_page.dart

```diff
@@ -79,6 +85,9 @@ class _ImageGridPageState extends State<ImageGridPage> {
           return SafeArea(
             child: CustomScrollView(
               slivers: [
+                SliverToBoxAdapter(
+                  child: Text(selectedFile != null ? "Selected:\n${selectedFile.path}" : ""),
```

Comment:
Would be good to add ValueListenableBuilder in this case of "selectedFile", so the screen only re-renders this item instead of the entire widget tree just to write different text path.

Reply:
this is just an example app so no need to put special care :)

Reply:
We don't use such a component in Vid app

Reply:
Makes sense!